pkg/operator/ceph/object/objectstore.go

```diff
@@ -38,6 +38,9 @@ const (
 	// AppName is the name Rook uses for the object store's application
 	AppName               = "rook-ceph-rgw"
 	bucketProvisionerName = "ceph.rook.io/bucket"
+
+	// An user with system privileges, used for dashboard service etc
+	AdminUser = "admin"
```

Comment:
Is this the ceph admin user? The dashboard admin user? The store admin user? This is unclear and could be made more clear by changing the name of the var to match.

Proper Go docs should be `// AdminUser is a user with...`

Reply:
Currently it is dashboard specific user, will change var name accordingly